plugins/scaffolder-node/report.api.md

```diff
@@ -442,6 +509,7 @@ export type TaskStatus =
 export type TemplateAction<
   TActionInput extends JsonObject = JsonObject,
   TActionOutput extends JsonObject = JsonObject,
+  TSchemaType extends 'v1' | 'v2' = 'v1',
```

Comment:
Would defaulting to `'v2'` work too? Makes it a bit trickier to use this one directly

Reply:
unfortunately not, as it looks like it breaks any usage today of just `const: TemplateAction<any, any> = createTemplateAction(...)` which seems to be in a lot of tests.

do you think it's worth doing it anyways?

Reply:
no I think it's alright to leave for when we remove v1